script/compile_resources.sh

```diff
@@ -3,6 +3,8 @@
 MB_SERVER_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")/../" && pwd)
 cd "$MB_SERVER_ROOT"
 
-./script/dbdefs_to_js.pl
+./script/dbdefs_to_js.pl --client
```

Comment:
Instead of storing the client/server DBDefs settings in different files, I had it generate a file containing only the client settings first, which is bundled in the gulp step, and then a file containing all the settings. The second file is used if the template renderer is run locally.